### PR TITLE
Bug 1647447 - Replace Push Health progess bar with donut graph

### LINKED ISSUE #6570: Commits / Progress area
Bring this into alignment with the UX mockup

https://bugzilla.mozilla.org/show_bug.cgi?id=1647447

### PR DESCRIPTION
**Pull request by SuyashSalampuria**
Fixes #6570 
replaced the progress bar with victory chart pie graph

### CODE REVIEW

**Comment by SuyashSalampuria** [comment]
@camd It is currently looking this
image

Please review this

**Comment by camd** [comment]
Screenshot 2020-07-13 09 09 52

**Comment by SuyashSalampuria** [comment]
The  best method I found was to use position: absolute 
It currently looks like this
Screenshot from 2020-07-14 00-13-04

**Comment by SuyashSalampuria** [comment]
@camd I have made the required changes
Please review them whether you get time:)

File: ui/push-health/Health.jsx [hunk 1]
```diff
@@ -229,17 +227,20 @@ export default class Health extends React.PureComponent {
           />
           {!!tests && !!currentRepo && (
             <React.Fragment>
-              <div>{percentComplete}% Complete</div>
-              <StatusProgress counts={status} />
+              <div className="d-flex mb-5">
+                <StatusProgress counts={status} />
+                <div>
```
Comment (camd) [inline ui/push-health/Health.jsx:232]: In the interest of centering the commit area around this a little better, let's add `className="mt-4 ml-2"` to this `<div>`

File: ui/shared/StatusProgress.jsx [hunk 2]
```diff
@@ -1,23 +1,41 @@
...
+        innerRadius={70}
+        radius={85}
+        style={{
+          labels: {
+            fontSize: 12,
+          },
+        }}
+      />
+      <div className="absolute">
+        <div style={{ font: '30px' }}>{percentComplete}%</div>
```
[the remark this answers is not in the document]
Comment (SuyashSalampuria) [reply ui/shared/StatusProgress.jsx]: {{ fontSize: '30px' }} is working for me. While using inline styling, we need double brackets.I have done the other changes although

**Comment by SuyashSalampuria** [comment]
image
It looks like this currently

**Pull request merged**